DOCSP-33948: Atomic typos and cleanup (#829)
mongodb/docs-node: source/fundamentals/stable-api.txt

   The {+stable-api+} feature requires MongoDB Server 5.0 or later.

>    Only use the {+stable-api+} feature if all the MongoDB

Reviewer: I: [only](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-may:~:text=Don%27t%20hyphenate.-,only,-Place%20only%20preceding)

```suggestion
   Use the {+stable-api+} feature only if all the MongoDB
```

.. tip::

>    To run commands using more than one version of the {+stable-api+},
>    instantiate a separate client with that version.

Reviewer: S: clarity

```suggestion
   You must create a new client for each version of the {+stable-api+} on
   which you need to run a command.
```

(I think this is accurate? ^^^^)

Author: seems right to me, changed!
